akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/JdbcProjectionImpl.scala

```diff
@@ -29,10 +29,56 @@ import akka.projection.internal.ProjectionSettings
 import akka.projection.internal.RestartBackoffSettings
 import akka.projection.internal.SettingsImpl
 import akka.projection.javadsl
-import akka.projection.jdbc.javadsl.JdbcSession
+import akka.projection.jdbc.JdbcSession
+import akka.projection.jdbc.scaladsl.JdbcHandler
+import akka.projection.scaladsl
+import akka.projection.scaladsl.Handler
 import akka.projection.scaladsl.SourceProvider
 import akka.stream.scaladsl.Source
 
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[projection] object JdbcProjectionImpl {
```

Comment:
Comparing with `CassandraRunningProjection` the `projectionState.abort` stuff is missing in `JdbcRunningProjection` (also ProjectionOffsetManagement but I guess that is coming in separate PR)

Reply:
yes, I will added those in other PRs